package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-grants-frontend",
-  "version": "1.46.0",
+  "version": "1.47.0",
```

Comment:
change version to 1.46.1

Reply:
its a new feature we use last digit after second decimal for bug fixes only